README.md

```diff
@@ -25,26 +25,26 @@ do not require that you build the library manually:
 
 - [Windows building](https://github.com/deltachat/deltachat-core/issues/306)
 
-If you can help with advancing or adding to these efforts, be our guest. 
+If you can help with advancing or adding to these efforts, be our guest.
```

Comment:
I know you didn't write this.  But "be our guest" sounds really offensive like "do what you want, we we'll never care or help".  Something like "please do contribute" would be much friendlier.